src/modules/multi-file-upload/module-usage.md

```diff
@@ -90,3 +96,51 @@ const attrs = {
         - **nameFieldIdentifier** _{String}_ - name field identifier
     - **locationMappings** _{Array}_ - list of file type to content type mappings based on a location identifier
     - **maxFileSize** {Number} - maximum file size allowed for uploading. It's a number of bytes.
+- **token** _{String}_ - CSRF token,
+- **siteaccess** _{String}_ - siteaccess identifier.
+
+**parentInfo** _{Object}_ - parent location meta information:
+
+- **contentTypeIdentifier** _{String}_ - content type identifier,
+- **contentTypeId** _{Number}_ - content type id,
+- **locationPath** _{String}_ - location path string,
+- **language** _{String}_ - language code identifier.
+
+### Optional props
+
+Optionally, Multi File Upload module can take a following list of props:
+
+**checkCanUpload** _{Function}_ - checks whether am uploaded file can be uploaded. The callback takes 4 params:
```

Comment:
`**checkCanUpload** _{Function}_ - checks whether the selected file can be uploaded. The callback takes 4 params:`
(uploaded -> selected)

Reply:
i'll stay with uploaded, because you don't select files, you're trying to upload files after selecting/dropping them.